wafer/templates/wafer/base.html

```diff
@@ -33,7 +33,7 @@ <h1>{{ WAFER_CONFERENCE_NAME }}</h1>
     </div>
     {% sponsors %}
     <script src="{% static 'vendor/jquery/dist/jquery.min.js' %}"></script>
-    <script src="{% static 'vendor/tether/dist/js/tether.min.js' %}"></script>
+    <script src="{% static 'node_modules/popper.js/dist/umd/popper.js' %}"></script>
```

Comment:
Shouldn't this be copied into `vendor/...` like the other JavaScript dependencies are?

Reply:
Doh, it should.